en/setup/dev_env_linux_arch.md

```diff
@@ -0,0 +1,48 @@
+# Development Environment on ArchLinux
+
+## Permissions
+
+The user needs to be added to the group "uucp":
+
+```sh
+sudo usermod -a -G uucp $USER
+```
+
+After that, logging out and logging back in is needed.
+
+> **Note** Log out and log in for changes to take effect! Also remove the device and plug it back in!
+
+## Script-based Installation
+
+Once ArchLinux is installed you can use our standard docker script 
+[archlinux_install_script.sh](https://github.com/PX4/containers/blob/master/docker/px4-dev/scripts/archlinux_install_script.sh)
+to install all dependencies required for building PX4 firmware.
+
+To install using this script, enter the following in a terminal:
+```sh
+wget https://raw.githubusercontent.com/PX4/containers/master/docker/px4-dev/scripts/archlinux_install_script.sh
+source ./archlinux_install_script.sh
```

Comment:
This needs sudo

Reply:
OK. I tried this again. It only works if I do
```
sudo -s
source ./archlinux_install_script.sh
```
Something to do with source being a shell command.

Anyway, will fix up docs appropriately.